java/gazelle/testdata/test_without_production_code/BUILD.out

```diff
@@ -0,0 +1 @@
+# gazelle:resolve java org.junit @maven//:junit_junit
```

Comment:
What the story with this addition?

Reply:
I do not recall it. Removed in the rebase